internal/archive/mta_location.go

```diff
@@ -60,6 +60,8 @@ type ITargetPath interface {
 // ITargetModule - Target Module interface
 type ITargetModule interface {
 	GetTargetModuleDir(moduleName string) string
+	GetTargetTmpDir() string
```

Comment:
Should this method still be defined in `ITargetModule`?

Reply:
removed